src/boundaries.rs

```diff
@@ -102,7 +102,55 @@ pub fn build_boundary(
     relation: &osmpbfreader::Relation,
     objects: &BTreeMap<osmpbfreader::OsmId, osmpbfreader::OsmObj>,
 ) -> Option<MultiPolygon<f64>> {
-    let roles = ["outer", "enclave", ""];
+    use geo::algorithm::contains::Contains;
+
+    let outer_polys = build_boundary_parts(relation, objects, vec!["outer", "enclave", ""]);
+    let inner_polys = build_boundary_parts(relation, objects, vec!["inner"]);
+    let mut multipoly = MultiPolygon(vec![]);
```

Comment:
it seems that this works (but for a test bug :confused: )

```rust
    if let Some(ref mut outers) = outer_polys {
        inner_polys.map(|inners| {
            inners.into_iter().for_each(|inner| {
                for ref mut outer in outers.0.iter_mut() {
                    if outer.contains(&inner) {
                        outer.interiors.push(inner.exterior);
                        break;
                    }
                }
            })
        });
    }
    outer_polys
```

Reply:
I've changed the test to avoid the edge case with contains.